### DISCUSSION TITLE
Unable to compile latest release?

### DISCUSSION THREAD

**Discussion by vijaypondini** [GriefPrevention/GriefPrevention#1753] [Q&A]
Hey there, i cloned the github repo https://github.com/TechFortress/GriefPrevention/ and when i cd'ed into it, and tried to build with `mvn clean install`, it gave me an error 

`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project GriefPrevention: Fatal error compiling: error: invalid target release: 16 -> [Help 1]`

Is there a jenkins or some other way to download the latest jars? As its not been updated in release of spigot/bukkit for a long time

P.S only reason i'm trying to get the latest is because it includes camotoy's work for making bedrock usernames with `.` to get /trust'ed with no problem

**Comment by Jikoo** [comment]
You need JDK 16 or higher installed to compile GP. Update Java.

Alternately, each successfully built commit has a little green checkbox next to it - click on it and select AppVeyor, then navigate to the artifacts tab to download.

**Comment by vijaypondini** [comment]
Nice, thank you! I'm unsure if Camotoy's work for allowing bedrock usernames with period to be exempt from permissions in /trust is merged into the master branch though?

**Reply by Jikoo** [reply]
If you check either the PR, the commit history, or the 16.18-RC1 patch notes you'll see that it has been.
